### ISSUE TITLE
to_pandas fails with Metrica Open Data (Game 3)

### ISSUE THREAD

**Issue by tim-hy** [PySport/kloppy#148]
The Metrica Open Data repository (https://github.com/metrica-sports/sample-data) recommends loading Game 3 through Kloppy. After attempting this locally, receive this error when running `to_pandas`:

![image](https://user-images.githubusercontent.com/99428477/182571646-519bdc24-07ac-4dd2-9040-f2cb453a17d9.png)

Unsure what is causing this. Cloud loading does not work either.

**Comment by koenvo** [comment]
Thanks for reporting @tim-hy !

Not sure yet why `player_data.coordinates` is `None` here. Let me check if it's an error at the Serializer side or at the source data.

**Comment by tim-hy** [comment]
Thanks for your assistance. After further inspection I have found that using `metrica.load_tracking_epts()` and `to_pandas()` to import an EPTS tracking file that I had generated myself loaded without error - therefore I think this is a dataset issue (to clarify, [this dataset](https://github.com/metrica-sports/sample-data/tree/master/data/Sample_Game_3)).

I'm not sure how to proceed here - perhaps open an issue on their repository?
